uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_manage-leave-requests.scss

```diff
@@ -57,20 +84,52 @@
  * @TODO remove this patch when Shoreditch theme is migrated to SSP
  */
 .chr_manage_leave_requests {
-  .btn.ui-select-toggle {
-    padding: 4px 10px !important;
-    text-transform: none !important;
+  &[data-role="admin"] {
+    .ui-select-toggle {
```

Comment:
@deb1990 

![image](https://user-images.githubusercontent.com/3973243/28061130-5cd7a420-6621-11e7-8790-5939c2150035.png)

It is fine in terms of layout, but it has a shadow which looks a bit inconsistent. Mine CSS rule removes the shadow. Shall I leave just a shadow rule or shall I remove rules for this selector completely?